rdr_service/alembic/versions/93fce807f225_fields_in_participant_summary_for_.py

```diff
@@ -0,0 +1,62 @@
+"""fields in participant summary for vaccine survey
+
+Revision ID: 93fce807f225
+Revises: 625fa91e6bac
+Create Date: 2021-05-25 10:54:56.877198
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import rdr_service.model.utils
+
+
+from rdr_service.participant_enums import PhysicalMeasurementsStatus, QuestionnaireStatus, OrderStatus
+from rdr_service.participant_enums import WithdrawalStatus, WithdrawalReason, SuspensionStatus, QuestionnaireDefinitionStatus
+from rdr_service.participant_enums import EnrollmentStatus, Race, SampleStatus, OrganizationType, BiobankOrderStatus
+from rdr_service.participant_enums import OrderShipmentTrackingStatus, OrderShipmentStatus
+from rdr_service.participant_enums import MetricSetType, MetricsKey, GenderIdentity
+from rdr_service.model.base import add_table_history_table, drop_table_history_table
+from rdr_service.model.code import CodeType
+from rdr_service.model.site_enums import SiteStatus, EnrollingStatus, DigitalSchedulingStatus, ObsoleteStatus
+
+# revision identifiers, used by Alembic.
+revision = '93fce807f225'
+down_revision = '625fa91e6bac'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('participant_summary', sa.Column('questionnaire_on_vaccine_minute_1', rdr_service.model.utils.Enum(QuestionnaireStatus), nullable=True))
```

Comment:
should this be cope_vaccine_minute_1?

Reply:
Do mean having the column names be `questionnaire_on_cope_vaccine_minute_1` and `questionnaire_on_cope_vaccine_minute_1_authored`?

Reply:
Yes.  Also, based on the codebook, I didn't think "minute" was part of the module name.  Not sure if prior precedent on these fields is to mirror the module code value string?

Reply:
I agree on COPE. I was trying to reduce the size of the name, but I don't think it's worth losing the COPE in the name.

I have minute in there because I've seen discussions of it referring to it as the "Minute Survey" without saying much about Covid or vaccine, and I thought it might be helpful for anyone trying to find the data if "minute" was there too. I'll try to track down what the messaging is between different partners for the name of the survey.

I could be wrong, but I didn't think we were necessarily basing these fields on the code names. Would consumers of the summary api (like HPRO and the sites) be aware of the code names?

Reply:
Added COPE. I'm still leaning toward leaving 'minute' there since Vibrent seems to be referring to it as the Minute Survey. But let me know what you think.

Reply:
I called it pdr_mod_cope_vaccine1 in the RDR 1.96.1 release PDR changes;  I don't necessarily think they have to align but I need to decide whether to change it in the hotfix.  Thoughts?

Reply:
I think they would align enough as they are, especially if they're for different audiences.